setup.py

```diff
@@ -32,7 +32,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.1',
+    version='0.5',
```

Comment:
We have the ancient code, that we might like to preserve and send up into pypi... that's 0.1
This could be 1.x, but not my call, it's just a big jump because the newer code will be a big jump.